apps/ltn/src/pages/design_ltn/turn_restrictions.rs

```diff
@@ -0,0 +1,334 @@
+use geom::Distance;
+use map_model::{RoadID, EditRoad, Map};
+use osm2streets::RestrictionType;
+use widgetry::mapspace::{World, WorldOutcome};
+use widgetry::{EventCtx, GeomBatch, Text, TextExt, Widget};
+use widgetry::Color;
+
+use super::{road_name, EditMode, EditOutcome, Obj};
+use crate::logic::turn_restrictions::{FocusedTurns, possible_destination_roads, restricted_destination_roads};
+use crate::render::{colors, render_turn_restrictions};
+use crate::{App, Neighbourhood};
+
```

Comment:
Please run `cargo fmt` too; there are some whitespace oddities that should get cleaned up